src/Fields/Fieldset.php

```diff
@@ -10,10 +10,13 @@
 
 class Fieldset
 {
-    protected $handle;
     protected $contents = [];
+    protected $handle;
+
+    /** @var bool */
+    protected $isAddonFieldset = false;
```

Comment:
Does it makes sense to make this naming a little less depending on `Addon`? I just think about a naming which describes what it does, rather than describe where it is saved. Maybe something like `isVirtualFieldset` or `isHiddenFieldset`? They could also be another prop in the fieldset markup: `hidden: true`. This way you could hide any fieldset, not only those which are in an addon directory.

Reply:
ya cuz it might not be an addon now, you could be extending elsewhere, good call.

Reply:
> They could also be another prop in the fieldset markup: hidden: true. This way you could hide any fieldset, not only those which are in an addon directory.

Very interesting idea, thanks!

Reply:
renamed